### PR TITLE
Add default series toggle in simple-chart

### PR DESCRIPTION
**Pull request by brandon-rusk**
## Changes

- Added new checkbox in Simple-chart wagtail block to toggle default state of series presentation

## How to test this PR

1. Pull PR
2. `yarn build`
3. `./runserver.sh`
4. Go to http://localhost:8000/admin/pages/4481/edit/
5. Create Simple-Chart block and test new checkbox
6. Publish/preview changes

## Screenshots

image

Unchecked:
image

Checked:
image

### CODE REVIEW

**Review by ielerol — COMMENTED** [review]
The wagtail checkbox and helper text lgtm

File: cfgov/templates/wagtailadmin/css/simple-chart-admin.css [hunk 1]
```diff
@@ -30,4 +34,9 @@
   .struct-block.simple-chart-block > .field[data-contentpath="download_file"] {
     margin-left: 0;
   }
-}
\ No newline at end of file
+
+  .struct-block.simple-chart-block > .field[data-contentpath="show_all_series_by_default"] {
+    /* margin-left: 0; */
```
Comment (wpears) [inline cfgov/templates/wagtailadmin/css/simple-chart-admin.css:39]:
```suggestion
```

✂️

File: cfgov/unprocessed/js/routes/on-demand/simple-chart/simple-chart.js [hunk 2]
```diff
@@ -170,6 +180,17 @@ function addProjectedMonths( chartObject, numMonths ) {
...
+     chartObject.series = chartObject.series.map( ( singluarSeries, i ) => {
+     // Skip the first series
+     if ( i > 0 ) {
+       singluarSeries.visible = false;
+     }
+     return singluarSeries;
+   } );
+   return chartObject;
+   } */
+
```
Comment (wpears) [inline cfgov/unprocessed/js/routes/on-demand/simple-chart/simple-chart.js:193]:
```suggestion
```
✂️

**Review by wpears — CHANGES_REQUESTED** [review]
👍 just the hanging comments to trim out

File: cfgov/templates/wagtailadmin/css/simple-chart-admin.css (hunk 1 above)
Comment (brandon-rusk) [reply cfgov/templates/wagtailadmin/css/simple-chart-admin.css]: Woops, thank you!

**Review by wpears — APPROVED** [review]

**Pull request merged**